controllers/api/index.js

```diff
@@ -2,15 +2,22 @@ const router = require('express').Router();
 const commentRoutes = require('./commentRoutes');
 const tripRoutes = require('./tripRoutes');
 const userRoutes = require('./userRoutes');
-const addTripRoutes = require('./addTripRoutes');
-const dailyRoutes = require('./dailyRoutes');
+
+// jw, I don't think we need this, since I already include this route in "tripRoutes.js", and that is why I am commenting out this.
+// also, I don't know what "dailyRoutes.js" is for.
+//const addTripRoutes = require('./addTripRoutes')
+//const dailyRoutes = require('./dailyRoutes');
+
 
 
 router.use('/comment', commentRoutes);
 router.use('/trip', tripRoutes);
 router.use('/users', userRoutes);
-router.use('/add-trip', addTripRoutes);
-router.use('/add-daily-log', dailyRoutes);
+
```

Comment:
These routes are needed for the user to add a daily log. I will add these again in a later commit.